pwsh/modules/TriggerAdoBuild/TriggerAdoBuild.psd1

```diff
@@ -0,0 +1,131 @@
+#
+# Module manifest for module 'TriggerAdoBuild'
+#
+# Generated by: seantreanor
+#
+# Generated on: 05/12/2023
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'TriggerAdoBuild'
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '37588431-8f25-4e80-9286-6c0a91e37cd6'
+
+# Author of this module
+Author = 'seantreanor'
```

Comment:
Do you want your PII on a public repository? :-)

Reply:
Updated